CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Remove `pytest-async` plugin #3846
 
 ### API-Changes
+- Do not allow missing documentation by default
```

Comment:
API-Changes in the sense we usually use it, are changes in the way cffi or jsonrpc are used; this is not changed by this pr.